mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -526,10 +530,10 @@ CLASS net/minecraft/unmapped/C_triydqro net/minecraft/block/AbstractBlock
 		METHOD m_yvamjply getBlock ()Lnet/minecraft/unmapped/C_mmxmpdoq;
 		METHOD m_yylpqepz onExplosion (Lnet/minecraft/unmapped/C_cdctfzbn;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_aahhrzpf;Ljava/util/function/BiConsumer;)V
 			ARG 3 explosion
-			ARG 4 biConsumer
+			ARG 4 dropConsumer
 		METHOD m_zntvpiao getLuminance ()I
 			COMMENT {@return the light level emitted by this block state}
-		METHOD m_zrpxtrfo isToolRequired ()Z
+		METHOD m_zrpxtrfo doesRequireTool ()Z
```

Comment:
don't like this change, I know it fits conventions better but it just sounds worse

Reply:
would you prefer to change `Settings` `#requiresTool` (method) -> `#toolRequired` or leave it as it was, an exception?

Reply:
both work for me, I would personally leave the `is` on it

Reply:
Oh the `Settings` one is a builder setter method, so it should match the field name
I'll update it to all be`toolRequired`/`isToolRequired`